odbc/odbc.md

```diff
@@ -0,0 +1,304 @@
+
+## **Installation on Windows**
+
+### **Driver Manager**
+
+Windows comes preinstalled with an ODBC driver manager. To access it, search for `Administrative Tools` on your system (either through the search bar, or `Control Panel > System and Security > AdministrativeTools`), and then from there select ODBC Data Sources (either 32-bit or 64-bit).
+
+From this application, you can set up your drivers.
+
+### **Driver**
+
+Now that you have the driver manager installed, you will have to install the ODBC driver that allows your IBM i machine to use unixODBC to talk to Db2. To get the driver, visit [the IBM i Access - Client Solutions page](https://www-01.ibm.com/support/docview.wss?uid=isg3T1026805) and select **Downloads for IBM i Access Client Solutions**. After logging in and redirected to the IBM I Access Client Solutions download page, scroll down and download the **ACS Windows App Pkg English (64bit)**.
+
+When the package has been downloaded and has been installed on your system, it should be available to see on your ODBC Data Source Administrator application.
+
+# **Configuration**
+
+With both the driver manager and driver installed on your system, you will still need to configure your drivers and DSNs (datasource names) to be able to use ODBC in your applications.
+
+* [Configuration with UnixODBC (IBM i, Linux)](#configuration-with-unixodbc-ibm-i-linux)
+* [Configuration on Windows](#configuration-on-windows)
+
+## **Configuration with UnixODBC (IBM i, Linux)**
+
+Both IBM i and Linux distributions use unixODBC and have nearly identical methods of setting up your drivers and your DSNs.
+
+### **`odbcinst.ini`**
+
+To declare a driver so that it can be referenced in your datasources, you will have to define your driver in a file called `odbcinst.ini`. This file can be found by entering the command 
+
+```
+$ odbcinst -j
+```
+
+Likely, this file is located in the default location, depending on whether you are on IBM i or Linux:
+
+* IBM i: `/QOpenSys/etc/odbcinst.ini`
+* Linux: `/etc/unixODBC/odbcinst.ini`
+
+In this file, you define a name for each of your drivers, and tell unixODBC where it can find driver referenced in your datasources. An example `odbcinst.ini` might look like:
+
+**`odbcinst.ini`**
+```ini
+[IBM i Access ODBC Driver]
+Description = IBM i Access ODBC Driver
+Driver      = /QOpenSys/pkgs/lib/libcwbodbc.so
+```
+(Note: You don't have to align the `=` signs, it just looks cleaner.)
+
+Luckily, the IBM i Access ODBC Driver will automatically create driver entries in your `odbcinst.ini` when you install the driver. When you define datasources in `odbc.ini` and `.odbc.ini` (covered below), you will use the name in the `[]` brackets as the name of the driver to use.
+
+If you have multiple drivers to define, you can simply create mutiple entries in the same file, as long as each driver is headed with a driver name in brackets, and no two drivers have the same name.
+
+**`odbc.ini` and `.odbc.ini`**
```

Comment:
I think this would be more beneficial to document like this:

- now that ODBC driver is installed, let's connect to the database
- ODBC uses a connection string with keywords, here's how to connect (show a DSN-less connection, eg `Driver=IBM i Access ODBC Driver;SYSTEM=foo;UID=bar;PWD=password;`)
- Show other configuration options, mention doc link to the list of keywords
- Passing all these options can get unwieldy, show how to create a DSN

Reply:
Changed in most recent push. Still have a tiny reference to odbcinst.ini in a note, because ODBC was confusing to me until I understood how all the files worked together, others might feel the same way.